### PR TITLE
i18n: Update Slovenian translation

### PR DESCRIPTION
**Pull request by matevz**
Includes latest ParaTime terms for deposits/withdrawals and some other minor fixes.

### CODE REVIEW

**Review by buberdds — APPROVED** [review]

File: src/locales/sl/translation.json [hunk 1]
```diff
@@ -25,38 +31,55 @@
...
-        "sent": "Delegiranih <Amount/> overitelju"
+        "received": "Prejeto <Amount> v delegiranje",
+        "sent": "Delegirano <Amount/> overitelju"
+      },
+      "consensusDeposit": {
+        "received": "Prejeto <Amount/> s pologom na ParaTime {{runtimeName}}",
+        "sent": "Polog <Amount/> na ParaTime {{runtimeName}}"
       },
+      "consensusWithdraw": {
+        "received": "Dvig <Amount/> s ParaTima {{runtimeName}}",
```
Comment (tjanez) [inline src/locales/sl/translation.json:60]: I think these `"received"` and `"send"` translation keys are not descriptive enough.
What is their meaning?

File: src/locales/sl/translation.json [hunk 2]
```diff
@@ -246,7 +280,7 @@
     "boundsNotSet": "Brez omejitev (0% - 100%)",
     "commission": "Provizija",
     "commissionBounds": {
-      "current": "<strong>{{lowerBound}}% - {{upperBound}}%</strong> ob trenutni epohi {{epoch}}",
+      "current": "<strong>{{lowerBound}} % - {{upperBound}} %</strong> v trenutni epohi {{epoch}}",
```
[the remark this answers is not in the document]
Comment (tjanez) [reply src/locales/sl/translation.json:283]: I also find it a bit less common to put a space between a number and the `%` symbol. Is there any special reason for it?

File: src/locales/sl/translation.json (hunk 1 above)
Comment (matevz) [reply src/locales/sl/translation.json:60]: They are used when listing transaction history of your account. The full path of the key is actually `transaction.consensusWithdraw.received`.

File: src/locales/sl/translation.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (matevz) [reply src/locales/sl/translation.json:283]: Hm, tricky question. So this is the only grammatically correct way of writing it in Slo. But it's probably similar to writing dates without the spaces between the day.month.year and other units like °C, km, h where people commonly write them without the space after the number, although it should be there.

[the remark this answers is not in the document]
Comment (matevz) [reply src/locales/sl/translation.json]: Decided to remove the space for now and we'll add it to all or none terms in Slo for consistency.

File: src/locales/sl/translation.json (hunk 1 above)
Comment (tjanez) [reply src/locales/sl/translation.json:60]: I understand that they are connected to listing an account's transaction history.

I meant to say that I don't find the `"received"` and `"sent"` very descriptive in terms of "this is the message to emit when an account is the receiving/sending account for a `consensus.Withdraw` transaction".

My ideas to make these more descriptive would be something like:
- `transaction.consensusWithdraw.sendingAccount`
- `transaction.consensusWithdraw.receivingAccount`

Or perhaps:
- `transaction.consensusWithdraw.sourceAccount`
- `transaction.consensusWithdraw.destinationAccount`

File: src/locales/sl/translation.json [hunk 3]
```diff
@@ -25,38 +31,55 @@
...
+        "received": "Prejeto <Amount> v delegiranje",
+        "sent": "Delegirano <Amount/> overitelju"
+      },
+      "consensusDeposit": {
+        "received": "Prejeto <Amount/> s pologom na ParaTime {{runtimeName}}",
+        "sent": "Polog <Amount/> na ParaTime {{runtimeName}}"
       },
+      "consensusWithdraw": {
+        "received": "Dvig <Amount/> s ParaTima {{runtimeName}}",
+        "sent": "Prejeto <Amount/> z dvigom s ParaTima {{runtimeName}}"
```
Comment (tjanez) [inline src/locales/sl/translation.json:61]: Perhaps simply:
`"Prejet dvig <Amount/> s ParaTima {{runtimeName}}"`

File: src/locales/sl/translation.json [hunk 4]
```diff
@@ -25,38 +31,55 @@
...
-      "loadingError": "Transakcij ni bilo mogoče pridobiti. Seznam je zato verjetno prazen ali zastarel.",
+      "loadingError": "Transakcij ni bilo mogoče pridobiti.",
       "reclaimEscrow": {
         "received": "Vračilo <Amount/> investitorju",
-        "sent": "Prejetih <Amount/> od overitelja"
+        "sent": "Prejeto <Amount/> od overitelja"
+      },
+      "stakingAllow": {
+        "received": "Prejeto dovoljenje za <Amount/>",
+        "sent": "Nastavi dovoljenje za <Amount/>"
```
[the remark this answers is not in the document]
Comment (matevz) [reply src/locales/sl/translation.json:76]: Good point. Maybe more descriptive:
`Pridobljeno dovoljenje za <Amount/>`
`Dano dovoljenje za <Amount/>`

File: src/locales/sl/translation.json [hunk 5]
```diff
@@ -25,38 +31,55 @@
...
     },
     "transaction": {
       "addEscrow": {
-        "received": "Prejeth <Amount/> v delegiranje",
-        "sent": "Delegiranih <Amount/> overitelju"
+        "received": "Prejeto <Amount> v delegiranje",
+        "sent": "Delegirano <Amount/> overitelju"
+      },
+      "consensusDeposit": {
+        "received": "Prejeto <Amount/> s pologom na ParaTime {{runtimeName}}",
```
[the remark this answers is not in the document]
Comment (matevz) [reply src/locales/sl/translation.json:56]: `ParaTime {{runtimeName}}` is grammatically correct in Slo (analogy Windows OS -> operacijski sistem Windows). I would stick to that.

File: src/locales/sl/translation.json (hunk 1 above)
Comment (matevz) [reply src/locales/sl/translation.json:60]: Let's move this to another PR since it's language-agnostic.

**Pull request merged**